### PR TITLE
feat(zoolander):rsweb-5487

### PR DESCRIPTION
**Pull request by amberRrucker**
https://jira.rax.io/browse/RSWEB-5487
rocket content type, split quote, random small things
(did some things im not proud of in order to prevent snowflake styling)

### CODE REVIEW

**Comment by aedile** [comment]
@amberRrucker - Rebase!

**Comment by bighappyface** [comment]
@amberRrucker would you mind adding a link to the O365 page on the examples page?

https://github.com/rackerlabs/zoolander/blob/492e39cc8c8600a9e620602aaa3eca0f477d0dda/styleguide/derek/examples/index.jade#L15-L18

File: styleguide/derek/examples/office365.jade [hunk 1]
```diff
@@ -0,0 +1,4 @@
+extends ../../_layouts/page
+
+block content
+  include ./_office365.html
```
Comment (bighappyface) [inline styleguide/derek/examples/office365.jade]: @amberRrucker you can add the `_mainnav` partial here as with solutions:

https://github.com/rackerlabs/zoolander/blob/492e39cc8c8600a9e620602aaa3eca0f477d0dda/styleguide/derek/examples/solutions.jade#L4

**Comment by amberRrucker** [comment]
@bighappyface added the link in there.

File: styleguide/derek/examples/office365.jade (hunk 1 above)
Comment (amberRrucker) [inline styleguide/derek/examples/office365.jade]: Added and pushed.

File: styleguide/derek/examples/_office365.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    </div>
+  </div>
+  <div class="row">
+      <div class="col-sm-4 center-block">
+          <button>Choose A Plan</button>
+      </div>
+  </div>
+</div>
+
+<nav class="navbar subnav">
```
Comment (bighappyface) [inline styleguide/derek/examples/_office365.html]: @amberRrucker sorry, I forgot to mention that you won't need this with the `_navbar` partial in play now. I am seeing double menus.

**Comment by bighappyface** [comment]
@amberRrucker please add an entry for `office365` to `_data.json`:

https://github.com/rackerlabs/zoolander/blob/492e39cc8c8600a9e620602aaa3eca0f477d0dda/styleguide/derek/examples/_data.json

**Comment by amberRrucker** [comment]
@bighappyface _data.json has been updated.

**Comment by bighappyface** [comment]
+1 looking good!

**Pull request merged**